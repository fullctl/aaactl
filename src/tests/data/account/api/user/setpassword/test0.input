{
  "password": "test",
  "password_new": "updatedpassword",
  "password_confirmation": "updatedpassword"
}
