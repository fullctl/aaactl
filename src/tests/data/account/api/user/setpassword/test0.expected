{"data": [{}], "errors": {}, "profiling": {"queries": 0}}
